cmd/collector/app/builder_flags.go

```diff
@@ -42,7 +41,6 @@ const (
 	collectorZipkinAllowedOrigins = "collector.zipkin.allowed-origins"
 	collectorZipkinAllowedHeaders = "collector.zipkin.allowed-headers"
 
-	collectorHTTPPortWarning       = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
 	collectorGRPCPortWarning       = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
 	collectorZipkinHTTPPortWarning = "(deprecated, will be removed after 2020-06-30 or in release v1.20.0, whichever is later)"
```

Comment:
I would prefer to remove all three of these in one PR, since they all affect similar flags

Reply:
I was thinking the same thing. Wasn't sure if they should be split for book keeping purposes but I'll go ahead and remove the other deprecated flags in this file.